package/kernel/mac80211/ath.mk

```diff
@@ -308,6 +310,11 @@ define KernelPackage/ath11k/config
                depends on PACKAGE_kmod-ath11k
                default y if TARGET_ipq807x
 
+       config ATH11K_MEM_PROFILE_512M
+               bool "Enable 512MB profile"
+               depends on PACKAGE_kmod-ath11k
+               default y if TARGET_ipq807x_generic_DEVICE_redmi_ax6 || TARGET_ipq807x_generic_DEVICE_xiaomi_ax3600
```

Comment:
I wouldn't call this the proper way to do it, it would make more sense to make a smallmem or whatever named ath11k-ahb variant and then include that in the device packages.